packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -127,6 +127,11 @@ class MenuContent extends React.Component {
     return { isToggleableMenu: this.isToggleable(), shouldReserveSpaceForIcon: this.shouldReserveSpaceForIcon() };
   }
 
+  componentDidMount() {
+    // Set focus to first focusable menu item
+    this.contentNode.querySelectorAll('li[tabindex="0"]')[0].focus();
```

Comment:
let's add data-attribute for menu items to access them instead of `tabindex`

Reply:
Changed to use class: https://github.com/cerner/terra-framework/pull/1626/commits/29718af367a1016afda71848e7e3d57acd5d957d

Reply:
Updated to use class along with tabindex (need first focusable item): https://github.com/cerner/terra-framework/pull/1626/commits/3249e13d48ecaa077e9c0a03ca16a6e5b8d557cf

Reply:
Used data-attribute for menu item,  updated in https://github.com/cerner/terra-framework/pull/1626/commits/0e944cfad9d2cb5c81725ab373c191e60248a11f